File: 조민준/2기/3주차/없는숫자더하기.java
// 0~9 까지의 숫자 중 배열에 없는 수를 더해서 리턴하는 문제
public class 없는숫자더하기 {
	class Solution {
	    public int solution(int[] numbers) {
	        int answer = 0;
	        
	        int n = numbers.length;
	        
	        for(int i = 0; i < n; i++) {
	            answer += numbers[i];	// answer에 배열 안에 있는 값들을 다 넣어준다.
	        }
	        
	        answer = 45 - answer;	// 0~9를 다 더하면 45이기 떄문에 45에서 배열 안의 값들을 빼면 결과 값 나옴
	        
	        return answer;
	    }
	}
}

File: 이준혁/2기/14주차/없는 숫자 더하기.js
function solution(numbers) {
    // 0부터 9까지가 배열의 변수로 들어감
    // 0~9를 다 더하면 45이므로
    // 45에서 현재 원소를 다 더한 값을 빼면 답
    let answer = 45; // 0~9의 합

    let numSum = 0;

    for (let i = 0; i < numbers.length; i++) {
        numSum += numbers[i];
    }

    answer -= numSum;

    return answer;
}
